classes/patreon_routing.php

```diff
@@ -133,9 +153,7 @@ function parse_request( &$wp ) {
 
 			}
 
-
 		}
-
 	}
 
```

Comment:
Let's be consistent about the number of blank lines here.. in this case we should probably just remove all of these gaps in the closing of this function

Reply:
Committed:

https://github.com/codebard/patreon-wordpress/commit/44c130ff6fc149bb2df226fb9c1383a2b74a63a8